domain/schema/controller.go

```diff
@@ -312,6 +313,25 @@ CREATE TABLE model_list (
 );`)
 }
 
+func modelMetadataSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE model_metadata (
```

Comment:
This is never going to be sparse compared to `model_list`. I think we should have a single table called `model`.